sample-tracking-status-overview-app/src/main/groovy/life/qbic/portal/sampletracking/components/projectoverview/ProjectOverviewViewModel.groovy

```diff
@@ -78,26 +70,36 @@ class ProjectOverviewViewModel {
     private void updateTotalSampleCount(StatusCount statusCount) {
         ProjectSummary summary = getProjectSummary(statusCount.projectCode)
         summary.totalSampleCount = statusCount.totalSampleCount
+
+        updateTotalSampleCount(statusCount)
```

Comment:
You are entering eternity here :D 
I would suggest to change that to 
```
private void updateTotalSampleCount(ProjectSummary projectSummary, int statusCount) {
    projectSummary.totalSampleCount = statusCount
}
```
this way you avoid the additional iteration over all projects that comes with `getProjectSummary`

Reply:
You could consider inlining this method as well since it is only one small assignment.